librecad/src/main/qc_applicationwindow.cpp

```diff
@@ -1493,7 +1493,7 @@ QC_MDIWindow* QC_ApplicationWindow::slotFileNew(RS_Document* doc) {
     if (w->getDocument()->rtti()==RS2::EntityBlock) {
         w->setWindowTitle(tr("Block '%1'").arg(((RS_Block*)(w->getDocument()))->getName()) + "[*]");
     } else {
-        w->setWindowTitle(tr("unnamed document %1").arg(id) + "[*]");
+        w->setWindowTitle(tr("Untitled Document %1").arg(id) + "[*]");
```

Comment:
If you're going to suggest a default filename as **Untitled** during save, you might as well use the same name as the default title.